assembly/assembly-main/src/assembly/bin/docker.sh

```diff
@@ -121,10 +121,6 @@ init() {
     export CHE_WORKSPACE_STORAGE_CREATE_FOLDERS=false
   fi
 
-  # Ensure that the user "user" has permissions for CHE_HOME and CHE_DATA
-  sudo chown -R user:user ${CHE_HOME}
-  sudo chown -R user:user ${CHE_DATA}
```

Comment:
@benoitf I think that these may be needed for the workspaces (that are still run as user `user`)